gapir/cc/interpreter.h

```diff
@@ -181,10 +181,10 @@ class Interpreter {
   const MemoryManager* mMemoryManager;
 
   // The builtin functions.
-  std::unordered_map<uint8_t, FunctionTable> mBuiltins;
+  FunctionTable mBuiltins[16];
 
   // The current renderer functions.
-  std::unordered_map<uint8_t, FunctionTable*> mRendererFunctions;
+  FunctionTable* mRendererFunctions[16] = {};
```

Comment:
Same here, maybe have a single comment for both `16` values, if you find a nice way to do this.

Reply:
Done